content/03.ml-concepts.md

```diff
@@ -22,6 +25,9 @@ Also, many conventional metrics for classification (e.g. area under the receiver
 Model performance should be evaluated with a carefully picked panel of relevant metrics that make minimal assumptions about the composition of the testing data [@doi:10.1021/acs.molpharmaceut.7b00578], with particular consideration given to metrics that are most directly applicable to the task at hand.
 
 Extreme cases warrant testing the robustness of the model and metrics on simulated data for which the ground truth is known.
-Said simulations can be used to verify the correctness of the model’s implementation as well.
+To accurately test the performance of the model, it is important that simulated datasets be generated for a range of parameters.
+Simulated datasets generated from parameters that are consistent with the assumptions of the deep learning models can be used to verify the correctness of the model’s implementation.
+Further varying the parameters to test variation and violations of model assumptions can further test the sensitivity of performance to those assumptions. 
```

Comment:
This sounds a bit repetitive: "Further varying the parameters to test variation"

Maybe:
```suggestion
Further modifying the parameters to test variation and violations of model assumptions can further test the sensitivity of performance to those assumptions. 
```